conferences/2025-agu.md

```diff
@@ -34,6 +34,103 @@ We're excited to hear what you have to say at our session!
 **See you in New Orleans!** 🚀
 
 
-## Invited speakers
+## Invited speakers: _TBD_
 
-_TBD_
+::::::{grid} 1 1 1 2
+
+:::{card}
+:header: **Invited Speaker - TBD!**
+
+_Check back soon!_
+:::
+
+:::{card}
+:header: **Invited Speaker - TBD!**
+
+_Check back soon!_
+:::
+
+::::::
+
+
+## Organizers
+
+::::::{grid} 1 1 1 2
+
+:::{card}
+:header: **Primary Convener - Qiusheng Wu**
+
+![Headshot of Dr. Qiusheng Wu](https://github.com/user-attachments/assets/f65928b0-1771-4bff-8331-a9168c16c9d8)
+
+**Dr. Qiusheng Wu** is an Associate Professor and the Director of Graduate Studies in the
+Department of Geography & Sustainability at the University of Tennessee, Knoxville. He
+also serves as an Amazon Scholar. Dr. Wu’s research focuses on geospatial data science
+and open-source software development, with an emphasis on leveraging big geospatial data
+and cloud computing to study environmental change, particularly surface water and
+wetland inundation dynamics. He is the creator of several widely used open-source Python
+packages, including geemap, leafmap, segment-geospatial, and geoai, which support
+advanced geospatial analysis and interactive visualization.
+
+His open-source work is available at [\@opengeos](https://github.com/opengeos).
+:::
+
+
+:::{card}
+:header: **Convener - Max Jones**
+
+_Check back soon!_
+:::
+
+
+:::{card}
+:header: **Convener - Chelle Gentemann**
+
+_Check back soon!_
+:::
+
+
+:::{card}
+:header: **Convener - Tyler Erickson**
+
+![Headshot of Dr. Tyler Erickson](https://media.licdn.com/dms/image/v2/D5603AQHLWPmrk6VHcQ/profile-displayphoto-shrink_400_400/profile-displayphoto-shrink_400_400/0/1726186563133?e=1757548800&v=beta&t=mNpM6EfWf76P6_O2HoeQfz9t0KWGeqO_ALHLnV6YqAw)
+
+**Dr. Tyler Erickson** is the founder of the geospatial consultancy VorGeo, a Technical
+Fellow at the non-profit Radiant Earth, and a founding editorial board member of the
+Cloud-Native Geospatial Forum. Previously he led the developer relations team for Google
+Earth Engine. Dr. Erickson's work focuses on building communities & ecosystems around
+emerging technologies, with an emphasis on technologies that can be leveraged by
+environmental sustainability projects.
+
+His writings & presentations can be found at <https://www.analyze.earth/>
+:::
+
+
+:::{card}
+:header: **Student Convener - Wilson Sauthoff**
```

Comment:
@wsauthoff I'm merging this PR so we can share out what we have so far. Please reply to this comment with a mini-bio and headshot and I'll open a new PR to integrate :)